earth_enterprise/src/fusion/portableglobe/servers/local/preview/setup.html

```diff
@@ -103,7 +104,8 @@
         Version: <span id="BuildVersion"></span>
         <br>
         <br>
-        &copy; 2010-2015 Google
+        &copy; 2010-2015 Google.
+        &copy; 2015-2018 the OpenGEE Contributors.
```

Comment:
Maybe we should stick a `<br>` between the two copyrights?  I'm assuming this is what is displayed in the portable server about text box.  Right now the two lines run together and wraps awkwardly.